infra/terraform/dev/deploy/launch.tf

```diff
@@ -0,0 +1,35 @@
+data "template_cloudinit_config" "config" {
+  gzip          = false
+  base64_encode = true
+
+  part {
+    content_type = "text/x-shellscript"
+    content      = <<EOT
+#!/bin/bash
+echo ECS_CLUSTER="wikijump-${var.environment}" >> /etc/ecs/ecs.config
+echo ECS_ENABLE_CONTAINER_METADATA=true >> /etc/ecs/ecs.config
+echo ECS_ENABLE_SPOT_INSTANCE_DRAINING=true >> /etc/ecs/ecs.config
+EOT
+  }
+
+  dynamic "part" {
+    for_each = var.user_data
+    content {
+      content_type = "text/x-shellscript"
+      content      = part.value
+    }
+  }
+}
```

Comment:
I don't really understand what this does. It sends that bash script to some new server?